source/codegen/metadata/nidmm/functions.py

```diff
@@ -1616,27 +1616,6 @@
         ],
         'returns': 'ViStatus'
     },
-    "InitExtCal": {
```

Comment:
I'd prefer marking the method as `codegen-method: none` over removing it. I assume your plan was to put this metadata back in in a future PR.

Reply:
Yes, okay

Reply:
Done adding.